### DISCUSSION TITLE
Summary of Tuning Feedback from Mount Hyjal Discord Channel

### DISCUSSION THREAD

**Discussion by AcetylGer** [chromiecraft/chromiecraft#7270] [Suggestions]
All credit goes to Haba on the CC Discord:

I've been lurking in this thread for a while and I believe there are a few core concerns that could be easily discussed and addressed if presented in a readable and actionable way.

1.) Tuning for newly released raids should be tuned for a full (24-25) group using very basic consumables and somewhat suboptimal compositions. In other words the average guild should find the content doable and engaging.

2.) Tuning should be "difficult" enough that correctly emulated mechanics (at the time of their expansion) are visible and executed in the fight. Accurate emulation should be a major goal of the project.

3.) However when in doubt, tuning should err on the side of "boring, easy and/or blizzlike." Bugs can unintentionally and drastically increase the difficulty of a fight and players will become bitter if they're unable (within reason) to complete a fight.

4.) Tuning should be public information. Without it being public it is difficult for the player base to provide adequate feedback and it creates unnecessary resentment in the playerbase who feel like they are in the dark.

5.) Tuning should be ongoing process especially after a major bug fix. Bug fixes can drastically change how a fight plays out and previous tuning may conflict with the new correct mechanics.

6.a) Players should not expect a bug free or perfect raiding experience on release. ChromieCraft is not Blizzard Entertainment. ChromieCraft is providing a free service with limited time and resources.

6.b) A three to four month release schedule is clearly not enough time to perfect a raid but is a necessary evil to create deadlines and push content along.

7.) Amicable, responsive and productive dialogue between staff and players is critical. There are a lot of people here who feel very passionate about this project and tuning in particular and more people should be added to the official tuning team who can help represent the server's interests.
8.) Mythic modes seem to have popular support so long as they DO NOT drop custom loot, DO NOT have custom mechanics and DO NOT consume development resources to the point where the release schedule is impacted. The mythic point system has been generally well received.

9.) Tuning reductions for older content as newer content is released also seems to have popular support. This point and the previous Point 8 seem to be in line with the server's interests and are conducive for long-term play and alts.

Let me know if I missed anything. Trying to be productive and helpful.

References: https://github.com/chromiecraft/PTR-review/issues/19

**Comment by Zelfie** [comment]
10/10. Good summary.
